README.md

```diff
@@ -236,6 +294,18 @@ type ElementProps = {
 };
 ```
 
+The props for the `PaymentRequestButtonElement` are:
```

Comment:
onFocus and onBlur are also available!

Should we make this type a union type of ElementProps and `{paymentRequest: ...}`?

Reply:
We could! It has no `onChange` prop, though.
The only reason I made this a separate component, instead of using the existing `Element` component, was to get more specific prop validation and types.

Reply:
ACK, makes sense.  Thinking about it more, maybe let's hold off on exposing more API surface area unless folks need it.